docs/Database_Schema.md

```diff
@@ -602,3 +602,144 @@ CREATE TABLE IF NOT EXISTS "account_history"(
+
+### _Create Script_
+```sql
+CREATE TABLE IF NOT EXISTS "proposal"(
+        "proposal_id"            TEXT        NOT NULL,
+        "block_height"           INTEGER     NOT NULL,
+        "tx_hash"                TINYBLOB    NOT NULL,
+        "app_name"               TEXT        NOT NULL,
+        "proposal_type"          INTEGER     NOT NULL,
+        "proposal_title"         TEXT        NOT NULL,
+        "vote_start_height"      INTEGER     NOT NULL,
+        "vote_end_height"        INTEGER     NOT NULL,
+        "doc_hash"               TINYBLOB    NOT NULL,
+        "fund_amount"            BIGINT(20)  NOT NULL,
+        "proposal_fee"           BIGINT(20)  NOT NULL,
+        "vote    _fee"           BIGINT(20)  NOT NULL,
+        "tx_hash_proposal_fee"   TINYBLOB    NOT NULL,
+        "proposer_address"       TEXT        NOT NULL,
+        "proposal_fee_address"   TEXT        NOT NULL,
+        "status"                 TEXT        NOT NULL,
+        "data_collection_status" TEXT        NOT NULL,
+    
+        PRIMARY KEY ("proposal_id(64)", "app_name(64)");
+);
+```
+----
+
+## 23. Table **proposal_metadata**
+
+### _Schema_
+
+| Column                     | Data Type | PK | Not NULL | Default  |Description|
+|:---------------------------|:--------- |:--:|:--------:| -------- | --------- |
+|  proposal_id               | TEXT      | Y  | Y        |          | The Proposal ID  |
+|  block_height              | INTEGER   |    | Y        |          | The block height  |
+|  voting_fee_hash           | TINYBLOB  |    | Y        |          | The hash of voting fee |
+|  vote_start_date           | INTEGER   |    | Y        |          | The vote start date of proposal |
+|  vote_end_date             | INTEGER   |    | Y        |          | The vote end date of proposal |
+|  detail                    | TEXT      |    | Y        |          | The Proposal detial  |
+|  submit_time               | INTEGER   |    | Y        |          | The Proposal submition time  |
+|  ave_pre_evaluation_score  | INTEGER   |    | Y        |          | The Proposal pre evaluation score  |
+|  pre_evaluation_start_time | INTEGER   |    | Y        |          | The Proposal pre evaluation start time |
+|  pre_evaluation_end_time   | INTEGER   |    | Y        |          | The Proposal pre evaluation end time  |
+|  proposer_name             | TEXT      |    | Y        |          | The proposer name |
+
+### _Create Script_
+```sql
+CREATE TABLE IF NOT EXISTS "proposal_metadata"(
+        "proposal_id"               TEXT      NOT NULL,
+        "voting_start_date"         INTEGER   NOT NULL,
+        "voting_end_date"           INTEGER   NOT NULL,
+        "voting_fee_hash"           TINYBLOB  NOT NULL,
+        "detail"                    TEXT      NOT NULL,
+        "submit_time"               INTEGER   NOT NULL,
+        "ave_pre_evaluation_score"  INTEGER   NOT NULL,
```

Comment:
assessResult items should be added. 

assessResult sample data
 "assessResult": {
    "average": 7,
    "nodeCount": 2,
    "completeness": 6,
    "realization": 6.5,
    "profitability": 7,
    "attractiveness": 7.5,
    "expansion": 8
  }

Reply:
applied